cljfmt/test/cljfmt/core_test.cljc

```diff
@@ -172,6 +172,25 @@
   (is (= (reformat-string "(foo)\n;bar\n;baz\n;qux\n(bang)")
          "(foo)\n;bar\n;baz\n;qux\n(bang)")))
 
+(deftest test-trailing-whitespace
+  (testing "trailing-whitespace"
+    (is (= (reformat-string "(foo bar) ")
+           "(foo bar)"))
+    (is (= (reformat-string "(foo bar)\n")
+           "(foo bar)"))
```

Comment:
Having a newline at the end of the file is valid, so `"(foo bar)\n"` shouldn't be changed.